config/v1/types_ingress.go

```diff
@@ -20,10 +20,12 @@ type Ingress struct {
 }
 
 type IngressSpec struct {
-	// default suffix.  It goes here or it gets removed from server
 }
 
 type IngressStatus struct {
+	// domain specifies the default subdomain that will be used to compose
```

Comment:
How about:

domain is used to generate the hostname for a route when none is specified.  The pattern that is used is: routename-projectname.domain.